admin_guide/configuring_gce.adoc

```diff
@@ -0,0 +1,64 @@
+= Configuring for GCE
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+OpenShift can be configured to access an link:https://cloud.google.com/compute/docs/disks/[GCE infrastructure], 
+including link:persistent_storage/persistent_storage_gce.html[using GCE volumes as persistent storage] for application data.
+After GCE is configured properly, some additional configurations will need to be completed on the OpenShift hosts.
+
+
+
```

Comment:
@adellape - that's correct, I accidentally left it there because I wasn't sure, but apparently it's not required for gce